src/formatters/default_formatter.erl

```diff
@@ -1254,11 +1304,12 @@ move_comments(Name, [Arg0 | Args]) ->
 is_qualified_function_composition(_, []) ->
     false;
 is_qualified_function_composition(Outside, [FirstArg | _]) ->
-    module_qualifier == erl_syntax:type(Outside) andalso
-        erl_syntax:type(FirstArg) == application andalso
-            module_qualifier ==
-                erl_syntax:type(
-                    erl_syntax:application_operator(FirstArg)).
+    erl_syntax:type(Outside) == module_qualifier
+    andalso erl_syntax:type(FirstArg) == application
+    andalso
```

Comment:
Why did it "break" in this second `andalso`?

Reply:
Because the right expression is one of those nasty qualified function compositions that forces everything around them to be indented. 😔